src/Http/Controllers/TeamspeakController.php

```diff
@@ -70,7 +76,7 @@ public function getConfiguration()
         $changelog = $parser->parse($this->getChangelog());
         
         return view('teamspeak::configuration', compact('tsUsername', 'tsPassword', 'tsHostname', 'tsServerQuery',
-            'tsServerPort', 'changelog', 'greenSettings'));
+            'tsServerPort', 'tsTags', 'changelog', 'greenSettings'));
```

Comment:
Instead, you could call `setting` directly in the blade where you need them. That will avoid the list of tool variable which start to be long :p

Reply:
Which would be awesome, however I don't know how to use a {{ setting... }} within an @if () statement in a blade, which is where I'm using these variables.